src/palera1n.ts

```diff
@@ -0,0 +1,121 @@
+const completionSpec: Fig.Spec = {
+  name: "palera1n",
+  description: "Jailbreaking tool for iOS/iPadOS 15.x-16.x arm64",
+  options: [
+    {
+      name: ["--version", ""],
+      description: "Print version",
+    },
+    {
+      name: ["--force-revert", ""],
+      description: "Remove jailbreak",
+    },
+    {
+      name: ["--setup-partial-fakefs", "-B"],
+      description: "Setup partial fakefs",
+    },
+    {
+      name: ["--setup-fakefs", "-c"],
+      description: "Setup fakefs",
+    },
+    {
+      name: ["--demote", "-d"],
+      description: "Demote",
+    },
+    {
+      name: ["--dfuhelper", "-D"],
+      description: "Exit after entering DFU",
+    },
+    {
+      name: ["--boot-args", "-e"],
+      description: "XNU boot arguments",
+      args: { name: "BOOT_ARGUMENTS" },
+    },
+    {
+      name: ["--enter-recovery", "-E"],
+      description: "Enter recovery mode",
+    },
+    {
+      name: ["--fakefs", "-f"],
+      description: "Boot fakefs",
+    },
+    {
+      name: ["--help", "-h"],
+      description: "Show help for palera1n",
+    },
+    {
+      name: ["--override-checkra1n", "-i"],
+      description: "Override checkra1n",
+      args: { name: "FILE" },
```

Comment:
Can you make this change for all the args that has a FILE type?
```suggestion
      args: { name: "FILE", suggestions: "filepaths" },
```